tools/multiqc/multiqc.xml

```diff
@@ -202,7 +202,10 @@ mkdir multiqc_WDir &&
         @LN_FILES@
     #elif str($repeat.software_cond.software) == "hicexplorer"
         #set $pattern = "Min rest. site distance"
-        @LN_FILES@
+        #for $i, $file in enumerate($repeat.software_cond.input)
+            grep -q "$pattern" $file || die "'$pattern' not found in the file" &&
```

Comment:
Can you replace the `grep` and the `ln` line there by
```
#set file_path = os.path.join($software_dir, str($i))
@CHECK_LN_FILE@
```
to reuse the already defined functions and also use a cleaner path joining

Reply:
`@CHECK_LN_FILE@` is not defined in the file and the execution of MultiQC quits with `cannot find 'identifier'`. I see that you use this identifier several times, probably I am doing something wrong?